tool/src/main/java/com/fidesmo/fdsm/Main.java

```diff
@@ -306,7 +306,7 @@ public static void main(String[] argv) {
                                 System.out.format("IIN: %s%n", HexUtils.bin2hex(iin));
                             System.out.format("OS type: %s (platform v%d)%n", platform, platformVersion);
 
-                            ensureBatched(bibo, client, fidesmoMetadata.get());
+                            fidesmoMetadata.get().ensureBatched(bibo, client, Optional.ofNullable(args.valueOf(OPT_TIMEOUT)),args.has(OPT_IGNORE_IMPLICIT_BATCHING),getCommandLineFormHandler());
```

Comment:
IMO Would be nice to refactor `Optional.ofNullable(args.valueOf(OPT_TIMEOUT))` and `args.has(OPT_IGNORE_IMPLICIT_BATCHING)` into some var/function to avoid duplication